sklearn/manifold/t_sne.py

```diff
@@ -754,7 +766,7 @@ def _fit(self, X, skip_num_points=0):
 
             # Find the nearest neighbors for every point
             knn = NearestNeighbors(algorithm='auto', n_neighbors=k,
-                                   metric=self.metric)
+                                   metric=self.metric, n_jobs=self.n_jobs)
```

Comment:
maybe pass `n_jobs=effective_n_jobs(self.n_jobs)` ?

Reply:
As it is called directly in the call to the `joblib` API, I would avoid it but as you want.